tests/unit/anchore_engine/services/policy_engine/policy/gates/test_passwd_file.py

```diff
@@ -0,0 +1,323 @@
+import pytest
+from anchore_engine.services.policy_engine.engine.policy.gates import passwd_file
+from anchore_engine.services.policy_engine.engine.policy.gate import ExecutionContext
+from anchore_engine.db.entities.policy_engine import Image, AnalysisArtifact
+
+
+image_id = "1"
+user = "admin"
+digest = "1"
+redis_p_entry = "redis:x:41:29:redis:/var/redis:/bin/zsh"
+foo_p_entry = "foo:x:45:45:foo:/var/foo:/bin/bash"
+
+
+@pytest.fixture()
+def create_image(monkeypatch):
```

Comment:
can this method be combined w/ the dockerfile gate method? `monkeypatch` can be an optional attribute?

Reply:
By assigning the image to the instance of `passwd_gate` and then accessing the image in the test through the gate as opposed to passing it in separately as a param?